docker/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Better to use `#!/usr/bin/env bash` shebang since we already use it in other scripts.

Reply:
ok no problem, I will change that.